src/commands/mod.rs

```diff
@@ -792,14 +792,14 @@ struct RebaseArgs {
     /// Each specified revision will become a direct child of the destination
     /// revision(s), even if some of the source revisions are descendants
     /// of others.
-    #[arg(long, short)]
+    #[arg(long, short, default_value = "@-")]
```

Comment:
Depending on how `clap` works, this could change the default more than we'd like. `jj rebase -d blah` should be equivalent to `jj rebase -b @ -d blah`. I'm not sure if we have a test for this (we should!), but I'm worried this will make the default be `jj rebase -s @- -d blah` (which is worse).

So, I wouldn't have a default for `-s`. I'm not sure why we don't have an explicit default for the `branch` option.

Reply:
I think this might, in fact, be breaking some tests.

I think the reason we didn't more explicitly specify that `--branch`'s default value is `@`, but the issue with that is that then `jj rebase -s` will complain that both `-s` and `-b` are set. I wonder if some use of `clap::ArgGroup` can make it clearer while still working correctly.

Reply:
You're right about this. git-branchless actually does not set the defaults directly (which I forget); it lets them be `None` and then does something sensible like you describe where the default is set for `-b` rather than `-s`, etc.